src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -50,6 +50,7 @@ type ControllerConfig struct {
 
 type Controller interface {
 	WaitAndUpdateNodesStatus()
+	ApproveCsrs(ctx context.Context, wg *sync.WaitGroup)
```

Comment:
one last comment an that's it:)
let's remove it for now, i don't want the changes in day1 code at all, the NewController returns object and not interface

Reply:
sure, removed from the interface.